config-verifier.yaml

```diff
@@ -3,6 +3,7 @@
 server:
   host: localhost
   port: 8080
+  jwt_secret: myjwtsecretphrase
```

Comment:
For admin auth I've put the secret into server.auth.jwt_secret and it's only relevant for the verifier in my case.
I guess it's ok to have 2 jwt secrets as admin api is not relevant to this one?